server/routes/serviceProviderReferrals/sessionStatusPresenter.test.ts

```diff
@@ -0,0 +1,36 @@
+import { SessionStatus } from '../../utils/sessionStatus'
+import SessionStatusPresenter from './sessionStatusPresenter'
+
+describe(SessionStatusPresenter, () => {
+  describe('text and tagClass', () => {
+    const expectedResults = [
+      {
+        status: SessionStatus.didNotAttend,
+        text: 'did not attend',
+        tagClass: 'govuk-tag--purple',
+      },
+      {
+        status: SessionStatus.scheduled,
+        text: 'scheduled',
+        tagClass: 'govuk-tag--blue',
+      },
+      {
+        status: SessionStatus.completed,
+        text: 'completed',
+        tagClass: 'govuk-tag--green',
+      },
+      {
+        status: SessionStatus.notScheduled,
+        text: 'not scheduled',
+        tagClass: 'govuk-tag--grey',
+      },
+    ]
+
+    expectedResults.forEach(({ status, text, tagClass }) => {
```

Comment:
Jest has [built-in support](https://jestjs.io/docs/api#testeachtablename-fn-timeout) for this kind of table testing - I'm not exactly sure what the difference is with writing the loop yourself, but might be worth a look?

Reply:
I didn't know about this, it's pretty neat.